PictureInput.vue

```diff
@@ -161,14 +176,16 @@ export default {
       }
     },
     onResize () {
-      let previewRatio = this.width / this.height
+      this.resize()
+    },
+    resize () {
+      let previewRatio = this.canvasWidth / this.canvasHeight
       let newWidth = this.$refs.container.clientWidth
-      if (newWidth === this.containerWidth) {
```

Comment:
Okay, this was to avoid a full canvas redraw if the window resize event didn't affect the canvas size, for the sake of efficiency. I suggest to reintroduce the check with a new condition:
` if (!this.rotatable && newWidth === this.containerWidth) { (...) }`